experimental/orchestrator/csharp_dotnetcore/02.todo-bot-with-adaptive-dialog/DialogExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Dialogs;
+
+namespace Microsoft.BotBuilderSamples
+{
+    public static class DialogExtensions
+    {
+        public static async Task Run(this Dialog dialog, ITurnContext turnContext, IStatePropertyAccessor<DialogState> accessor, CancellationToken cancellationToken = default(CancellationToken))
+        {
```

Comment:
@vishwacsena do we need this Run() as opposed to [DialogExtensions.RunAsync()](https://github.com/microsoft/botbuilder-dotnet/blob/54cc02627e4d0ffbe8ae68ea3f534bdc7aae4386/libraries/Microsoft.Bot.Builder.Dialogs/DialogExtensions.cs#L31) in the shipped packages?

Reply:
Not sure. But would like to keep it this way for preview 1. This is consistent with the current adaptive samples.

Reply:
> Not sure. But would like to keep it this way for preview 1. This is consistent with the current adaptive samples.

Sounds good to me